tests/pytests/unit/modules/test_slackware_service.py

```diff
@@ -0,0 +1,251 @@
+"""
+    :codeauthor: Piter Punk <piterpunk@slackware.com>
+"""
+# Import python libs
+import os
+
+import pytest
+
+# Import Salt Libs
+import salt.modules.slackware_service as slackware_service
+
+# Import Salt Testing Libs
+from tests.support.mock import MagicMock, patch
+
+glob_output = [
```

Comment:
thanks for all these updates, really appreciate them. I have just one more thing. Can we add these variables to a pytest fixture? This actually helps speed up things when pytest is collecting tests and prevents any issues we have ran into the past where someone edited a list/dict at the global module level that affects other tests.

Reply:
It took a while, but following your advice and with the explanations of the @waynew from the Saltstack Test Clinic, I removed all the global variables and took the opportunity to make a big overhaul of test names, descriptions, comments and reorganized some them. I hope this time everything is fine :)